tool/tsh/db.go

```diff
@@ -207,27 +198,77 @@ func databaseLogout(tc *client.TeleportClient, db tlsca.RouteToDatabase) error {
 
 // onDatabaseEnv handles "tsh db env" command.
 func onDatabaseEnv(cf *CLIConf) {
+	tc, err := makeClient(cf, false)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	database, err := pickActiveDatabase(cf)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	env, err := dbprofile.Env(tc, *database)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	for k, v := range env {
+		fmt.Printf("export %v=%v\n", k, v)
+	}
+}
+
+// onDatabaseConfig handles "tsh db config" command.
+func onDatabaseConfig(cf *CLIConf) {
+	tc, err := makeClient(cf, false)
+	if err != nil {
+		utils.FatalError(err)
+	}
 	profile, err := client.StatusCurrent("", cf.Proxy)
 	if err != nil {
 		utils.FatalError(err)
 	}
+	database, err := pickActiveDatabase(cf)
+	if err != nil {
+		utils.FatalError(err)
+	}
+	host, port := tc.WebProxyHostPort()
+	fmt.Printf(`Name:      %v
+Host:      %v
```

Comment:
If would consider adding a format option for this to make this machine-parsable - this way it can be converted to any other format on the fly without much trouble.

Reply:
This is intended to be human-readable first. I agree it might make sense to add other format in future, I'll leave it as a potential improvements if anyone actually needs it.